### PR TITLE
#167891584 Notifications for new travel request(User opt-in or opt-out in-app notification)

### PR DESCRIPTION
**Pull request by Pomile**
#### What does this PR do?
Have an admin user opt-in or opt-out of in-app notification

#### Description of Task to be completed?
- [x] Write unit test
- [x] Write documentation
- [x] Reuse the helper function that updates the user record
- [x] Create a method that opt-in or opt-out a user in-app notification

##### Unit test
1. Clone the project repository
2. Cd into the project repository
3. Run `git pull`
4. Run `git checkout  ft-inAppNotification-configOpt-endpoint-167891584`
5. Run `npm install`
6. Create database `database_name`
7. Create .env file
8. Run npm test

##### Development test with Postman
 1. Run `npm run devstart`
 2. Launch postman
 3. Create a new patch request with  `/api/v1/notifications/in_app_opt` 
 4. Add a request body `{ inAppOpt: true }`
 5. Add authorization(token) in the request header.
 6. Hit send

#### What are the relevant pivotal tracker stories?
[#167891584](https://www.pivotaltracker.com/story/show/167891584)
#### Any background context you want to add?
Nil
#### Screenshots
Nil

### CODE REVIEW

File: src/controllers/NotificationOpt.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        inAppOpt: updatedUser.inAppOpt
+      }
+    );
+
+    return res.status(response.code).send(response);
+
+  }
+}
+
+export default NotificationOpt;
```
Comment (thatkazuk1) [inline src/controllers/NotificationOpt.js]: Add a newline here

Comment (Pomile) [reply src/controllers/NotificationOpt.js]: Thank you Demond.

Comment (thatkazuk1) [reply src/controllers/NotificationOpt.js]: *Desmond* man.

Comment (Pomile) [reply src/controllers/NotificationOpt.js]: Noted Thanks

**Review by daniellamarr — APPROVED** [review]

**Pull request merged**